src/components/cards/BiobankReportCard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="mg-biobank-card container">
+    <script v-html='jsonld' type='application/ld+json' />
```

Comment:
I'm not sure what you are trying to achieve here? Can you elaborate

Reply:
Basically, the task is to add Bioschemas for Biobanks and Collections. This embeds the json-ld bioschemas representation of the biobank in BiobankReportCard. The schema is taken from the `jsonld` variable which is computed in  mapBiobankDataCatalog function

Reply:
That I understand, but what is the purpose of having this embedded on the page within script tags, that is what i mean

Reply:
Bioschemas.org suggests using json-ld format and embedding it in the web page. https://bioschemas.org/tutorials/what_why_schema

Reply:
thank you! For your explanation